### DISCUSSION TITLE
Export Return Period Plot

### DISCUSSION THREAD

**Discussion by geohazard89** [CLIMADA-project/climada_python#772] [General]
Is there any option to export this plot to geotiff? 

![image](https://github.com/CLIMADA-project/climada_python/assets/63537357/8f7f8a4e-11c3-40ba-b0fb-c7b303d231ac)

**Comment by chahank** [comment]
There is no dedicated function for this. Please use the existing methods for i/o from the matplotlib package.

**Comment by peanutfun** [comment]
@geohazard89 We do not support file output via GeoTIFFs. One of the issues is that GeoTIFFs are rasterized per definition, whereas Climada operates on a set of "unrelated" geographic points in a geopandas GeoDataFrame format. However, if the data is rasterized, we store this information in the `meta` attribute, in your case `haz.centroids.meta`. You could use this information to define your raster. For writing data to GeoTIFF, I suggest you consult SO, e.g. https://stackoverflow.com/q/66383026.

@chahank Matplotlib is unable to write GeoTIFFs.

I will move this thread to the discussions, since writing GeoTIFFs from Climada data structures is not supported.
